fe/fe-core/src/main/java/com/starrocks/scheduler/PartitionBasedMvRefreshProcessor.java

```diff
@@ -360,39 +359,48 @@ public void filterPartitionByRefreshNumber(Set<String> partitionsToRefresh, Mate
         if (partitionRefreshNumber <= 0) {
```

Comment:
Do we need to consider getAutoRefreshPartitionsLimit in `getPartitionNamesByRangeWithPartitionLimit` too?

Reply:
it's already considered ? I don't understand